src/java/htsjdk/samtools/BamIndexValidator.java

```diff
@@ -84,4 +85,70 @@ public static int exhaustivelyTestIndex(final SamReader reader) { // throws Exce
         return 0;
     }
 
+    /**
+     * A less time-consuming index validation that only looks at the first and last references in the index
+     *
+     * @param reader
+     * @return # of chunks examined, or 0 if there is no browseable index for the reader
+     */
+    public static int lessExhaustivelyTestIndex(final SamReader reader) {
+        // look at all chunk offsets in a linear index to make sure they are valid
+
+        if (reader.indexing().hasBrowseableIndex()) {
+
+            // content is from an existing bai file
+            final CachingBAMFileIndex existingIndex = (CachingBAMFileIndex) reader.indexing().getBrowseableIndex();
+            final int n_ref = existingIndex.getNumberOfReferences();
+
+

```

Comment:
I feel like Nils now, but there's an extra newline here.  And on lines 126 and 145.  :(